CMakeLists.txt

```diff
@@ -45,7 +45,7 @@ IF( "${CMAKE_VERSION}" VERSION_EQUAL "3.1" OR
   CMAKE_POLICY(VERSION 3.1.0)
 ENDIF()
 
-IF(POLICY CMP0037)
+IF("${CMAKE_VERSION}" VERSION_LESS "3.11" AND POLICY CMP0037)
   # allow to override "test" target for quick tests
   CMAKE_POLICY(SET CMP0037 OLD)
```

Comment:
how will this be done in the future? Use `make quick_tests` instead of `make test`?

Reply:
Still `make test`. CMake 3.11 got less stressed about custom target names again. Basically, as long as we do not call `ENABLE_TESTING()` in the main `CMakeLists.txt` file, we can simply define a target `test`. So my solution is to simply set the policy only for everything that is older and add an additional guard "if target is not already defined" around the `package` and `tests` target.